website/docs/docs/core/connect-data-platform/profiles.yml.md

```diff
@@ -33,7 +33,6 @@ This section identifies the parts of your `profiles.yml` that aren't specific to
   [static_parser](/reference/global-configs/parsing): <true | false>
   [cache_selected_only](/reference/global-configs/cache): <true | false>
   [printer_width](/reference/global-configs/print-output#printer-width): <integer>
-  [log_format](/reference/global-configs/logs): <text | json | default>
```

Comment:
Please see line 28 for the first instance of `log_format` in this yml code block.